spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -88,23 +90,35 @@
           ],
           "dates": [{"date": "1-1-2021", "type": "other", "description": "When the item was originally created."}],
           "formats": "application/pdf",
-          "locations": [{"place": "Boston, Massachusetts, United States"}, {"place": "East Peoria, Illinois, United States"}]
-        },
+          "locations": [{"place": "Boston, Massachusetts, United States"}, {"place": "East Peoria, Illinois, United States"}],
+          "funding": [{
+            "funder": {
+              "name": "National Library of Medicine (NLM)",
+              "identifier": "0060t0j89",
+              "scheme": "ror"
+            },
+            "award": {
+              "title": "",
+              "number": "F37 LM009568 ",
+              "identifier": "",
+              "scheme": ""
+            }}]},
         "provenance": {
           "created_by": {
             "user": assistant.username
           },
           "on_behalf_of": {
             "user": user.username
           }
-        }
+        },
       },
       "file": {
         "filename": generic_file.filename,
         "content_path": nil # there is no file/content attached with this factory made GewnericFile
       }
     }.to_json
   end
+  let(:invenio_rdm_record_converter) { described_class.new(generic_file) }
```

Comment:
Putting this in a let variable so we can reuse a single instance for multiple specs.